rules/docker-sonic-mgmt-framework.mk

```diff
@@ -28,6 +28,7 @@ $(DOCKER_MGMT_FRAMEWORK)_CONTAINER_NAME = mgmt-framework
 $(DOCKER_MGMT_FRAMEWORK)_RUN_OPT += --privileged -t
 $(DOCKER_MGMT_FRAMEWORK)_RUN_OPT += -v /etc/sonic:/etc/sonic:ro
 $(DOCKER_MGMT_FRAMEWORK)_RUN_OPT += -v /etc:/host_etc:ro
+$(DOCKER_MGMT_FRAMEWORK)_RUN_OPT += -v /var/run/dbus:/var/run/dbus:rw
```

Comment:
`/var/run/dbus` will exist regardless, it's only to ensure that the D-Bus communications within the container are sent to the host. If `ENABLE_HOST_SERVICE=n`, then there won't be any listener for the `org.SONiC.HostService` address, and therefore, calls to the host would fail anyway.